### PR TITLE
feat: Bicep Module for Bing Resource

### PR DESCRIPTION
**Pull request by tanujbhatia1708**
## Description

If you haven't already, read the full [contribution guide](https://github.com/Azure/bicep-registry-modules/blob/main/CONTRIBUTING.md). The guide may have changed since the last time you read it, so please double-check. Once you are done and ready to submit your PR, edit the PR description and run through the relevant checklist below.

Enable GitHub Worksflows in your fork to enable auto-generation of assets with our [GitHub Action](/.github/workflows/push-auto-generate.yml).
To trigger GitHub Actions after auto-generation, [add a GitHub PAT](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token) as a secret in your forked repository called `PAT`.

## Adding a new module

- [x] A proposal has been submitted and approved.
- [x] I have included "Closes #{module_proposal_issue_number}" in the PR description.
- [x] I have run `brm validate` locally to verify the module files.
- [x] I have run deployment tests locally to ensure the module is deployable.

## Updating an existing module

- [ ] This is a bug fix:
  - [ ] Someone has opened a bug report issue, and I have included "Closes #{[bug_report_issue_number](https://github.com/Azure/bicep-registry-modules/issues/401)}" in the PR description.
  - [ ] The bug was found by the module author, and no one has opened an issue to report it yet.
- [ ] I have run `brm validate` locally to verify the module files.
- [ ] I have run deployment tests locally to ensure the module is deployable.
- [ ] I have read the [Updating an existing module](https://github.com/Azure/bicep-registry-modules/blob/main/CONTRIBUTING.md#updating-an-existing-module) section in the contributing guide and updated the `version.json` file properly:
  - [ ] The PR contains backwards compatible bug fixes, and I have NOT bumped the MAJOR or MINOR version in `version.json`.
  - [ ] The PR contains backwards compatible feature updates, and I have bumped the MINOR version in `version.json`.
  - [ ] The PR contains breaking changes, and I have bumped the MAJOR version in `version.json`.
- [ ] I have updated the examples in README with the latest module version number.

### CODE REVIEW

**Comment by tanujbhatia1708** [comment]
@dciborow  @shenglol  Can we please register the subscription for Microsoft.Bing ?

**Comment by tanujbhatia1708** [comment]
@dciborow @shenglol Please help to review

**Comment by shenglol** [comment]
Bing Search is no longer part of Cognitive Services since October 2020. I wonder if we should use `ai/bing-search` as the module path.

**Comment by tanujbhatia1708** [comment]
@shenglol  i have updated the path . Thanks

**Review by dciborow — APPROVED** [review]

File: modules/ai/bing-resource/README.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  }
+}
+```
+
+### Example 2
+
+Deploy a Bing Custom Search resource with the standard SKU
+
+```
+module bing-search-resource 'br/public:ai/bing-resource:0.0.1' = {
```
Comment (dciborow) [inline modules/ai/bing-resource/README.md:52]:
```suggestion
module bing-search-resource 'br/public:ai/bing-resource:1.0.1' = {
```

File: modules/ai/bing-resource/README.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+| `endpoint` | `string` | Bing Endpoint   |
+
+## Examples
+
+### Example 1
+
+Deploy a Bing Search v7 resource with the free SKU
+
+```
+module bing-search-resource 'br/public:ai/bing-resource:0.0.1' = {
```
Comment (dciborow) [inline modules/ai/bing-resource/README.md:36]:
```suggestion
module bing-search-resource 'br/public:ai/bing-resource:1.0.1' = {
```

**Pull request merged**